nsxt/resource_nsxt_policy_ipsec_vpn_service.go

```diff
@@ -102,15 +123,21 @@ func resourceNsxtPolicyIPSecVpnServiceImport(d *schema.ResourceData, m interface
 	s := strings.Split(importID, "/")
 	err := fmt.Errorf("Expected policy path for the IPSec VPN Service, got %s", importID)
 	// The policy path of IPSec VPN Service should be like /infra/tier-0s/aaa/locale-services/bbb/ipsec-vpn-services/ccc
-	if len(s) != 8 {
+	// or /infra/tier-0s/aaa/ipsec-vpn-services/bbb
+	if len(s) != 8 && len(s) != 6 {
 		return nil, err
 	}
-	d.SetId(s[7])
+	useLocaleService := len(s) == 8
+	d.SetId(s[len(s)-1])
 	s = strings.Split(importID, "/ipsec-vpn-services/")
 	if len(s) != 2 {
 		return []*schema.ResourceData{d}, err
 	}
-	d.Set("locale_service_path", s[0])
+	if useLocaleService {
+		d.Set("locale_service_path", s[0])
+	} else {
+		d.Set("gateway_path", s[0])
+	}
 	return []*schema.ResourceData{d}, nil
 }
 
```

Comment:
looks like new support is missing in `deleteNsxtPolicyIPSecVpnService` below

Reply:
yes, seems it's also missing in getNsxtPolicyIPSecVpnService, now fixed